core/src/main/scala/cromwell/core/eventbus/PubSubMediator.scala

```diff
@@ -0,0 +1,48 @@
+package cromwell.core.eventbus
+
+import akka.actor.{ActorRef, ActorSystem}
+
+final case class EventMsg[A](topic: A, payload: Any)
+
+/**
+  * This is a wrapper to EventStream as one of the event bus with SubChannel trait.
+  */
+trait PubSubMediator {
```

Comment:
My reading of the email was that the pub/sub mediator is no longer the primary way of passing control messages between engine and backend - but rather a logging and tracking mechanism. 

If that's a correct and true reading, I think I'd lobby for focusing this PR just on the engine/backend interface and adding the pub/sub mediator in another PR focussed on that? If I've missed the point of the email, maybe a clarification? Thanks!